File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums_seq = []
        longest = 0

        for num in nums:
            if((num-1) not in nums):
                seq_length= 1
                curr_num = num
                while((curr_num+1) in nums):
                    curr_num += 1
                    seq_length += 1
                
                longest = max(longest, seq_length)
            else:
                continue


        return longest

File: Data Structures & Algorithms/maximum-subarray/submission-4.cs
public class Solution {
    public int MaxSubArray(int[] nums) {
        int res = nums[0];
        int currSum = nums[0];

        for (int i = 1; i < nums.Length; i++) {
            currSum = Math.Max(nums[i], currSum + nums[i]);
            res = Math.Max(res, currSum);
        }

        return res;
    }
}
